mmv1/third_party/terraform/website/docs/d/container_cluster.html.markdown

```diff
@@ -2,59 +2,1092 @@
 subcategory: "Kubernetes (Container) Engine"
 page_title: "Google: google_container_cluster"
 description: |-
-  Get info about a Google Kubernetes Engine cluster.
```

Comment:
Just caught that you are overwriting the datasource markdown file instead of the resource markdown file. Please undo the changes to this file and add your field documentation to the one under `docs/r/`